dev/docs/builtins-reference.md

```diff
@@ -318,7 +320,47 @@ slicefinder(X,W, y, k, paq, S);
 ### Usage
 ```r
 X = rand (rows = 50, cols = 10)
-y = X %*% rand(rows=ncol(X), 1)
+y = X %*% rand(rows = ncol(X), cols = 1)
 w = lm(X = X, y = y)
 ress = slicefinder(X = X,W = w, Y = y,  k = 5, paq = 1, S = 2);
 ```
+
+## `confusionMatrix`-Function
+
+A `confusionMatrix` is a technique for summarizing the performance of a classification algorithm.
+Calculating a confusion matrix can give you a better idea of what your classification model is getting right and what types of errors it is making.
```

Comment:
`you`, `your` if possible can be avoided, instead. So, it is (without these two words!)
`a confusion matrix can give a better idea of what the classification model is getting right.`